tests/PKSim.Tests/Presentation/IndividualSettingsPresenterSpecs.cs

```diff
@@ -16,6 +16,7 @@
 using PKSim.Presentation.Presenters.DiseaseStates;
 using PKSim.Presentation.Presenters.Individuals;
 using PKSim.Presentation.Views.Individuals;
+using DiseaseState = PKSim.Core.Model.DiseaseState;
```

Comment:
no code changes but one additional `using `- why?

Reply:
It's because I have a DiseaseState snapshot as well that is probably conflicting now. So I need to specify which to use